recipes/telescope/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "telescope" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True # [py27]
+  script: "python setup.py install --single-version-externally-managed --record=record.txt"
+
+about:
```

Comment:
Please make sure the structure of your recipe is `package`, `source`, `build`, `requirements`, `test`, `about`, `extras`. Different structure can cause issues with the nightly uploads.